modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -1434,6 +1433,34 @@ return view.extend({
 				o.password = true;
 
 				/* extra RADIUS settings start */
+				var attr_validate = function(section_id, value) {
+					if (!value)
+						return true;
+
+					if (!/^[0-9]+(:s:.+|:d:[0-9]+|:x:([0-9a-zA-Z]{2})+)?$/.test(value) )
+						return _('Must be in %s format.').format('<attr_id>[:<syntax:value>]');
+
+					return true;
+				};
+
+				var req_attr_syntax = _('Format:') + '<code>&ltattr_id&gt[:&ltsyntax:value&gt]</code>' + '<br />' +
```

Comment:
Please terminate the `&lt;` and `&gt;` entities with semicolons.

Reply:
🤦